src/one_dragon/base/config/yaml_config.py

```
@@ -32,9 +38,12 @@ def __init__(self,
         self._copy_from_sample: bool = copy_from_sample
         """配置文件不存在时 是否从sample文件中读取"""
 
+        self._read_sample_only: bool = read_sample_only
+        """是否只读取sample文件（即使.yml文件存在也只读sample）"""
```

Ruff (0.15.9)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)